app/stylesheets/layout.scss

```diff
@@ -30,7 +30,7 @@ body {
   left: 0;
   position: absolute;
   right: 0;
-  top: $header-height + $filters-closed-height;
+  top: $header-height;
```

Comment:
I added it back but it didn't make a difference on my end in the dashboard page. I think it could be because of where we placed the filters (outside the header).